packages/contracts/contracts/L1/SecurityCouncil.sol

```diff
@@ -0,0 +1,121 @@
+     *
+     * @param transactionId Index of the submitted transaction.
+     * @param outputRoot The output root.
+     * @param l2BlockNumber The L2 block number of the output root.
+     */
+    event Validation(
+        uint indexed transactionId,
+        bytes32 outputRoot,
+        uint128 l2BlockNumber
+    );
+
+    /**
+     * @notice The address of the colosseum contract. Can be updated via upgrade.
+     */
+    address public immutable COLOSSEUM;
+
+    /**
+     * @notice Represents a l2 output data.
+     *
+     * @custom:field outputRoot    Root of the L2 output this was proven against.
+     * @custom:field timestamp     Timestamp at whcih the withdrawal was proven.
+     * @custom:field l2OutputIndex Index of the output this was proven against.
+     */
+    struct OutputInfo {
+        bytes32 outputRoot;
+        uint128 l2BlockNumber;
+    }
+
+    /**
+     * @notice Disallow calls from anyone except Colosseum.
+     */
+    modifier onlyColosseum() {
+        require(msg.sender == COLOSSEUM, "SecurityCouncil: only the colosseum contract can be sender");
+        _;
+    }
+
+    /**
+     * @custom:semver 0.1.0
+     *
+     * @param _colosseum Address of the Colosseum contract.
+     */
+    constructor(address _colosseum) Semver(0, 1, 0) {
+        COLOSSEUM = _colosseum;
+    }
+
+    /**
+     * @notice Initializer.
+     */
+    function initialize(
+        address[] memory _owners,
+        uint _numConfirmationsRequired
+    ) public override initializer {
+        super.initialize(_owners, _numConfirmationsRequired);
+    }
+
+    /**
+    * @notice Allows Colosseum to request for validate output data.
+    *
+    * @param _outputRoot Output root byte data
+    * @param _l2BlockNumber l2 block number corresponding to output
+    * @param _destination address that calldata call to
+    * @param _data calldata for callback purpose
+    * @return Returns transaction ID.
+    */
+    function requestValidation(
+        bytes32 _outputRoot,
+        uint128 _l2BlockNumber,
+        address _destination,
+        bytes memory _data
+    )
+        public
+        onlyColosseum
+        returns (uint)
+    {
+        outputInfos[transactionCount] = OutputInfo({
+            outputRoot: _outputRoot,
+            l2BlockNumber: _l2BlockNumber
+        });
+        emit Validation(transactionCount, _outputRoot, _l2BlockNumber);
+        return submitTransaction(_destination, 0, _data);
+    }
+
+    /**
+    * @notice Allows an owner to submit and confirm a transaction.
+    *
+    * @param _destination Transaction target address.
+    * @param _value Transaction ether value.
+    * @param _data Transaction data payload.
+    * @return Returns transaction ID.
+    */
+    function submitTransaction(address _destination, uint _value, bytes memory _data)
+        public
+        override
+        onlyColosseum
+        returns (uint)
+    {
+        require(_value == 0, "SecurityCouncil: deposit transaction not allowed");
```

Comment:
Is the deposit transaction when value is zero?

Reply:
We don't allow deposit transactions here.